.gitignore

```diff
@@ -15,3 +15,6 @@ notes.md
 notes/
 examples/
 jenkins/
+
+.idea/*
```

Comment:
Is this folder specific for this project?  If you have your own files that you want to be gitignored I suggest using a ~/.gitignore_global.

Reply:
This folder `.idea` is added by pyCharm.